lib/rack/turnout.rb

```diff
@@ -14,9 +14,14 @@ def initialize(app, config={})
 
   def call(env)
     request = Turnout::Request.new(env)
-    settings = Turnout::MaintenanceFile.find
+    mf = Turnout::MaintenanceConfig.find
+    if mf
+      settings = mf
+    else
+      settings = Turnout::MaintenanceConfig.default
+    end
```

Comment:
This is a nitpick, I know, but I prefer this style of assignment with a conditional
```ruby
settings = if mf
  mf
else
   Turnout::MaintenanceConfig.default
end
```

Or really, it's simple enough that a ternary operator would probably be even better
```ruby
settings = mf ? mf :  Turnout::MaintenanceConfig.default
```